languages/python/exercises/concept/conditionals/.docs/instructions.md

```diff
@@ -0,0 +1,77 @@
+<!-- 
+Date: 26 Jan 2021
+This is a draft to see if this is question theme is suitable or not and will change according to 
+review feedback
+ -->
+
+In this exercise we will develop a simple control system for a nuclear reactor.
+
+In order for reactor to produce power it must be in a state of criticality. 
+If the the reactor becomes less than criticality it can damage the reactor. 
+If it goes beyond criticalty it can result in a meltdown.
+
+[//]: # 'Check for criticality'
```

Comment:
What is this used for? `[//]: # 'Check for criticality'`

Reply:
I was following the convention used in one of the concept exercise called elyses-enchantments instruction.md in the python track